CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,548 @@
+---
+CIP: ?
+Title: On-chain dApp Certification Metadata
+Category: Metadata
+Status: Proposed
+Authors:
+    - Romain Soulat <romain.soulat@iohk.io>
+    - Nicolas Jeannerod <nicolas.jeannerod@tweag.io>
+    - Mathieu Montin <mathieu.montin@tweag.io>
+    - M. Ali Modiri <mixaxim@gimbalabs.io>
+    - Simon Thompson <simon.thompson@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/499
+    - https://github.com/input-output-hk/Certification-working-group/pull/18
+    - https://docs.google.com/document/d/1vC4cS-gMLIvgduSh5Fipn_lU_0Ou0NY4X_a9lWJz2B8
+Created: 2023-02-23
+License: CC-BY-4.0
+---
+
+
+## Abstract
+
+Certification is the process of providing various kinds of assurance of DApps through the provision of verifiable, trustworthy metadata linking to relevant evidence of assurance, such as audit reports, test run data, formal verification proofs etc.
+
+Currently, certification does not have a standardised way to be stored and presented to all stakeholders (DApp developer, DApp stores, end-users, auditors, ...) in an immutable, persistent and verifiable way.
+
+This CIP descibes a standardised method for certificates to be published and stored on-chain and for stakeholders to be able to verify the different claims of this certificate.
+
+This CIP describes how a certification certificate can be registered on-chain and how to anchor it to a more detailed off-chain certification report.
+
+## Motivation: why is this CIP necessary?
+
+It is expected that evidence of various kinds of assurance of DApps is recorded in an immutable and verifiable way on the Cardano blockchain.
+Three level of certications are thought of at the time of this CIP: level 1 to 3. They were presented at the following [link](https://iohk.io/en/blog/posts/2021/10/25/new-certification-levels-for-smart-contracts-on-cardano/)
```

Comment:
@RSoulatIOHK (cc @Ryun1) are you going to move these into a separate CIP on certification levels?  It would make sense to have them defined in something more rigorous than a blog entry almost 2 years old.

Editorially I think it would be fine to leave the certification levels defined here, since as I understand no other CIP refers to them.  They mention the blog entry as a source and are practically confirmed here.  

But please @RSoulatIOHK & other authors let us know if you are going to be producing that "certification levels" CIP while this one is being reviewed... since, if so, those definitions should be in the new CIP rather than in this one.  RSVP 🧐

Reply:
Resolved: addressed by https://github.com/cardano-foundation/CIPs/pull/499/commits/520e8f7d1ac96982959d0dd769b4ba9e8e7eb508 (now in `Path to Active`)